library_generation/test/generate_pr_description_unit_tests.py

```diff
@@ -38,36 +49,41 @@ def test_get_commit_messages_current_is_older_raise_exception(self):
             True,
         )
 
-    def test_get_commit_messages_current_and_baseline_are_same_raise_exception(self):
+    def test_get_commit_messages_current_and_baseline_are_returns_empty_message(self):
```

Comment:
nit: Shouldn't this be "get commit messages with _equal/same_ current and baseline commit"?